packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -637,6 +638,13 @@ class SemanticsData with Diagnosticable {
   /// be set when [maxValueLength] is set.
   final int? currentValueLength;
 
+  /// The URI that this node links to.
+  ///
+  /// See also:
+  ///
+  /// * [SemanticsFlag.isLink], which indicates that this node is a link.
+  final String? linkUri;
```

Comment:
any reason not to use `Uri` ?

Reply:
In c++, it has to be a string anyway. But I can make it a `Uri` here and convert it to a string later?

Reply:
I think using Uri will be better

Reply:
I also think it would be easier for both implementer and developer to leave this as a String since that's how it's represented on the engine side

Reply:
I generally found out that it will be easier for the developer to use uri for public facing API. Otherwise you will likely to see a lot of 
```dart
Semantics(
  linkUri: Uri.parse(...)..replaceQueryParameters(...).toString()
)
```
It also give use flexibility to decide what is sent to engine and how. for example do we want to normalize path or queryparameters.

Reply:
Good point. Changing now